### DISCUSSION TITLE
favicon not showing on macos safari

### DISCUSSION THREAD

**Discussion by stardvst** [vercel/next.js#65390] [Help] [answered]
### Summary

i'm following this tutorial: https://youtu.be/cuzw4vL1z5E?t=1179
at the linked time he copies the favicon.ico to app/ directory and it shows the picture in browser tab.
on mac, it doesn't work. i googled and found out i need to add apple-icon.png and icon.png, so i did: https://nextjs.org/docs/app/api-reference/file-conventions/metadata/app-icons
i have deleted .next folder and rebuild the app, but it still doesn't show the icon in tab, how can i fix it?

### Example

https://github.com/stardvst/travel-app

**Comment by icyJoseph** [comment, marked as answer]
Often you have to hard-reload the page, https://help.codehs.com/en/articles/4951972-how-to-clear-your-browser-cache-and-hard-refresh

**Reply by stardvst** [reply]
that worked! thank you.
